cmd/ui/src/views/Home/Home.tsx

```diff
@@ -22,3 +22,7 @@ const Home: React.FC = () => {
 };
 
 export default Home;
+
+// TODO: Remove before merge to fix linter messages
+let arr = [].reduce(function(prev: never, curr: never, idx: number, arr: never[]) {})
```

Comment:
This will survive merge. It will act as a control change for the follow-up PR where we fix existing issues